packages/cli/src/databases/sqlite/migrations/1636626154932-CreateUserManagement.ts

```diff
@@ -1,78 +1,96 @@
 
-		await queryRunner.query(
-			'CREATE TABLE "settings" ("key"	TEXT NOT NULL,"value"	TEXT NOT NULL DEFAULT \'\',"loadOnStartup"	boolean NOT NULL default false,PRIMARY KEY("key"));',
 		);
 
 		await queryRunner.query(
-			`CREATE INDEX "IDX_xeendlvptc5jy4hbol17b5xery" ON "execution_entity" ("workflowId") `,
+			`CREATE TABLE "${tablePrefix}settings" ("key"	TEXT NOT NULL,"value"	TEXT NOT NULL DEFAULT \'\',"loadOnStartup"	boolean NOT NULL default false,PRIMARY KEY("key"));`,
 		);
 
 		// Insert initial roles
 		await queryRunner.query(`
-			INSERT INTO role (name, scope)
+			INSERT INTO "${tablePrefix}role" (name, scope)
 			VALUES ("owner", "global");
 		`);
 
-		const insertedOwnerRole = await queryRunner.query('SELECT last_insert_rowid() as insertId');
+		const instanceOwnerRole = await queryRunner.query('SELECT last_insert_rowid() as insertId');
 
 		await queryRunner.query(`
-			INSERT INTO role (name, scope)
+			INSERT INTO "${tablePrefix}role" (name, scope)
 			VALUES ("member", "global");
 		`);
 
 		await queryRunner.query(`
-			INSERT INTO role (name, scope)
+			INSERT INTO "${tablePrefix}role" (name, scope)
 			VALUES ("owner", "workflow");
 		`);
 
+		const workflowOwnerRole = await queryRunner.query('SELECT last_insert_rowid() as insertId');
+
 		await queryRunner.query(`
-			INSERT INTO role (name, scope)
+			INSERT INTO "${tablePrefix}role" (name, scope)
 			VALUES ("owner", "credential");
 		`);
 
+		const credentialOwnerRole = await queryRunner.query('SELECT last_insert_rowid() as insertId');
+
+		const survey = loadSurveyFromDisk();
+
+		const ownerUserId = uuid();
+		await queryRunner.query(`
+			INSERT INTO "${tablePrefix}user" (id, globalRoleId, personalizationAnswers) values
+			(?, ?, ?)
+		`, [ownerUserId, instanceOwnerRole[0].insertId, survey ?? null]);
+
+		await queryRunner.query(`
+			INSERT INTO "${tablePrefix}shared_workflow" (createdAt, updatedAt, roleId, userId, workflowId)
+			select DATETIME('now'), DATETIME('now'), '${workflowOwnerRole[0].insertId}', '${ownerUserId}', id from "${tablePrefix}workflow_entity"
+		`);
+
 		await queryRunner.query(`
-			INSERT INTO user(id, globalRoleId) values
-			('${uuid()}', ${insertedOwnerRole[0].insertId})
+			INSERT INTO "${tablePrefix}shared_credentials" (createdAt, updatedAt, roleId, userId, credentialsId)
+			select DATETIME('now'), DATETIME('now'), '${credentialOwnerRole[0].insertId}', '${ownerUserId}', id from "${tablePrefix}credentials_entity"
 		`);
 
 		await queryRunner.query(`
-			INSERT INTO settings(key, value, loadOnStartup) values
+			INSERT INTO "${tablePrefix}settings" (key, value, loadOnStartup) values
 			('userManagement.hasOwner', 'false', true)
 		`);
```

Comment:
we need to add to index the `workflowId` on executions:

`CREATE INDEX "IDX_xeendlvptc5jy4hbol17b5xery" ON "execution_entity" ("workflowId") `,

Reply:
Not doing this here since it's covered by https://github.com/n8n-io/n8n/pull/2794